test/integration/testdata/packagecollision/interface.go

```diff
@@ -3,8 +3,10 @@ package packagecollision
 import (
 	afoo "github.com/Makpoc/gomigen/test/integration/testdata/packagecollision/a/foo"
 	bfoo "github.com/Makpoc/gomigen/test/integration/testdata/packagecollision/b/foo"
+	. "github.com/Makpoc/gomigen/test/integration/testdata/packagecollision/dotimported/foo"
+	barfoo "github.com/Makpoc/gomigen/test/integration/testdata/packagecollision/renamedpackage/bar"
```

Comment:
I think the alias is defeating the purpose here.
I think you should leave it as `"github.com/Makpoc/gomigen/test/integration/testdata/packagecollision/renamedpackage/bar"` and use it as `foo.Foo` below.

Reply:
I tried that. gofmt (or goimports) didn't let me leave it unaliased. I had to save it with a text editor, which may prove hard to maintain in the future, but we'll see.